CSharpHTTPClient/Client.cs

```diff
@@ -296,21 +302,23 @@ public override bool TryInvokeMember(InvokeMemberBinder binder, object[] args, o
         /// </summary>
         /// <param name="client">Client object ready for communication with API</param>
         /// <param name="request">The parameters for the API call</param>
+        /// <param name="cancellationToken">A token that allows cancellation of the http request</param>
         /// <returns>Response object</returns>
-        public async virtual Task<Response> MakeRequest(HttpClient client, HttpRequestMessage request)
+        public async virtual Task<Response> MakeRequest(HttpClient client, HttpRequestMessage request, CancellationToken cancellationToken)
         {
-            HttpResponseMessage response = await client.SendAsync(request).ConfigureAwait(false);
+            HttpResponseMessage response = await client.SendAsync(request, cancellationToken).ConfigureAwait(false);
             return new Response(response.StatusCode, response.Content, response.Headers);
         }
 
         /// <summary>
         ///     Prepare for async call to the API server
         /// </summary>
         /// <param name="method">HTTP verb</param>
+        /// <param name="cancellationToken">A token that allows cancellation of the http request</param>
         /// <param name="requestBody">JSON formatted string</param>
         /// <param name="queryParams">JSON formatted queary paramaters</param>
         /// <returns>Response object</returns>
-        private async Task<Response> RequestAsync(string method, string requestBody = null, string queryParams = null)
+        private async Task<Response> RequestAsync(string method, CancellationToken cancellationToken, string requestBody = null, string queryParams = null)
```

Comment:
Please move the `cancellationToken` parameter to the end for consistency with our v9 SDK.